### PR TITLE
Fixed changing label color on canvas

### LINKED ISSUE #4966: Shape color in canvas is not changed after objects label selection
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour
When user changed labels for drawn object in objects panel, shape color on canvas should changed according selected labels color.

### Current Behaviour
Only new label color applying in background of object card in object pane, not on shape in image canvas. New color applied when canvas size changed.

### Possible Solution
I think that there are event handler missed somewhere that do update shape color on canvas after label selection.

### Steps to Reproduce (for bugs)

1. Draw any shape on image in task with more that two labels with different colors
1. Change label of drown object to different one
1. See that color of drawn shape still representing old label selection not current one.

### Context
It's hard to validate correctness of applying labels changes on frame without these visual feedback.

### Your Environment

- Git hash commit (`git log -1`): fe148d492b19c9ec35a95c1cb2927d360fd20675
- Docker version `docker version` (e.g. Docker 17.0.05): 20.10.12
- Are you using Docker Swarm or Kubernetes? no
- Operating System and version (e.g. Linux, Windows, MacOS): Ubuntu 22.04.1
- Browser: Vivaldi 5.4 (Chromium 104 based)

This bug is also reproduces in app.cvat.ai website.

### PR DESCRIPTION
**Pull request by bsekachev**
### Motivation and context
Resolved #4966 

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: cvat-canvas/src/typescript/canvasView.ts [hunk 1]
```diff
@@ -1884,6 +1884,15 @@ export class CanvasViewImpl implements CanvasView, Listener {
                 }
             }
 
+            if (drawnState.label.id !== state.label.id || drawnState.color !== state.color) {
```
[the remark this answers is not in the document]
Comment (bsekachev) [reply cvat-canvas/src/typescript/canvasView.ts:1887]: @klakhov 
When ``colorBy`` is set to ``Instance`` and we change object color (``enter`` shortcut, or via the object menu)

[the remark this answers is not in the document]
Comment (klakhov) [reply cvat-canvas/src/typescript/canvasView.ts:1887]: ok, i see

**Review by klakhov — APPROVED** [review]

**Comment by sebastiansimyonghwee** [comment]
Color is still not changed for Points, when using shortcut key to change the label.

**Pull request merged**